.moban.yaml

```diff
@@ -43,13 +43,15 @@ dependencies:
 gitignore_extra_rulesets:
   - node
 
+requires:
+  - https://gitlab.com/coala/mobans.git
 configuration:
   template_dir:
     - .moban.dt/
-    - ../coala-mobans/templates/
-    - ../coala-mobans/assets/
+    - mobans:templates/
+    - mobans:assets/
```

Comment:
Needs to be in quotes.
Forward slash may be removed.

Reply:
I doesnt need to be in quotes (otherwise the mobans job would fail), and it is more clear what it is when it has a trailing `/`

Reply:
👍 I was referring to https://github.com/moremoban/moban/tree/master/docs/level-10-moban-dependency-as-git-repo